pkg/agent/openflow/pipeline.go

```diff
@@ -632,9 +639,15 @@ func (f *featurePodConnectivity) podUplinkClassifierFlows(dstMAC net.HardwareAdd
 		nonVLAN = false
 	}
 	for _, ipProtocol := range f.ipProtocols {
+		table := ClassifierTable
+		priority := priorityHigh
+		if ipProtocol == binding.ProtocolIPv6 {
+			table = IPv6Table
+			priority = priorityNormal
```

Comment:
Why priority is different for IPv6?

Reply:
The table is different (ClassifierTable vs IPv6Table), thus the new flows' priority depends on the existing flows' priority.

Reply:
A follow up question - why the table is different for IPv6? Could you add some comments to explain this?

@wenyingd : could you review the flow design?

Reply:
Added comments.
Antrea uses a separate table for IPv6, thus we can save these flows to IPv6 table. We can also add a new table for IPv4, but this requires some flow design change, so I prefer to keep existing IPv4 design.

Reply:
I did not see you added the comment to code?

Reply:
> I did not see you added the comment to code?

Added in [Line 644](https://github.com/antrea-io/antrea/pull/4008/files#diff-120d06ea0809e1c6293ce9a1f6b2f4bf37d4fba0ec569be913f884aa8cdd4124R644)

Reply:
> Why not use `ClassifierTable` to classify IPv6 packet directly?

IPv6 NDP traffic is handled by IPv6Table priorityHigh, and we need to add flows after NDP related flows. Thus we use priorityNormal in IPv6Table.
For IPv4, ARP is processed earlier than ClassifierTable.

Reply:
Considering the IPv6 NDP packets received from uplink or host-interface, I think we can move IPv6Table before `ClassifierTable`, then the packets we processed in `ClassifierTable` and the following tables are focusing on the general IPv6 traffic, and we could keep the pipeline design the same between IPv4 and IPv6.